addons/l10n_it/data/template/account.fiscal.position-it.csv

```diff
@@ -20,4 +20,49 @@
 "split_payment_fiscal_position","Split Payment","5","0","0",,,"22v","22vsp_group","Scissione dei Pagamenti","Operations subject to split payment – the seller does not collect the VAT pursuant to ex art.17-ter of the D.P.R. 633/1972, the buyer is obliged to pay the Revenue Agency.","Operazione soggetta a split payment – il cedente non incassa l’Iva ai sensi dell’ex art.17-ter del D.P.R. 633/1972, l’acquirente è obbligato al versamento all’Agenzia delle Entrate."
 "","","","","","","","10v","10vsp_group","","",""
 "","","","","","","","5v","5vsp_group","","",""
-"","","","","","","","4v","4vsp_group","","",""
\ No newline at end of file
+"","","","","","","","4v","4vsp_group","","",""
+"scrap_fiscal_position","Scrap materials - RC","6","0","0",,,"22am","22rc_n61","Materiali di Scarto - IC","Operation subject to Reverse Charge and exempt from stamp duty – the seller does not collect the VAT pursuant to ex art.74, comma 7-8, D.P.R. 633/1972","Operazione soggetta a reverse charge ed esente da imposta da bollo – il cedente non incassa l’Iva ai sensi dell’ex art.74, comma 7-8, D.P.R. 633/1972."
+"","","","","","","","10am","10rc_n61","","",""
+"","","","","","","","5am","5rc_n61","","",""
+"","","","","","","","4am","4rc_n61","","",""
+"","","","","","","","00v","0rc_n61","","",""
+"gold_fiscal_position","Industrial Gold - RC","7","0","0",,,"22am","22rc_n62","Oro Industriale - IC","Operation subject to Reverse Charge and exempt from stamp duty – the seller does not collect the VAT pursuant to ex art.17, comma 5, D.P.R. 633/1972","Operazione soggetta a reverse charge ed esente da imposta da bollo – il cedente non incassa l’Iva ai sensi dell’ex art.17, comma 5, D.P.R. 633/1972."
+"","","","","","","","10am","10rc_n62","","",""
+"","","","","","","","5am","5rc_n62","","",""
+"","","","","","","","4am","4rc_n62","","",""
+"","","","","","","","00v","0rc_n62","","",""
+"gold_inv_fiscal_position","Gold Investments - RC","8","0","0",,,"22as","22rc_n62b","Investimenti in Oro - IC","Operation subject to Reverse Charge and exempt from stamp duty – the seller does not collect the VAT pursuant to ex art.17, comma 5, D.P.R. 633/1972","Operazione soggetta a reverse charge ed esente da imposta da bollo – il cedente non incassa l’Iva ai sensi dell’ex art.17, comma 5, D.P.R. 633/1972."
+"","","","","","","","10as","10rc_n62b","","",""
+"","","","","","","","5as","5rc_n62b","","",""
+"","","","","","","","4as","4rc_n62b","","",""
+"","","","","","","","00v","0rc_n62b","","",""
+"construction_fiscal_position","Construction Subcontractors - RC","9","0","0",,,"22as","22rc_n63","Subappalto Edilizia - IC","Operation subject to Reverse Charge and exempt from stamp duty – the seller does not collect the VAT pursuant to ex art.17, comma 6, letter a, DPR 633/1972","Operazione soggetta a reverse charge ed esente da imposta da bollo – il cedente non incassa l’Iva ai sensi dell’ex art. 17, comma 6, lettera a, DPR 633/1972."
```

Comment:
@lordkrandel Talking to @tsb-odoo , this should be the only fiscal position kept that we are adding here.  It is the only one where the logic will apply to the entire invoice (and as such it is in handy for the user to put it) most similar to the co-contractant fiscal position in Belgium, while in the other cases it is exceptions per line.  (which we could e.g. tackle by putting the law reference in the PDF on every line, but that can be done later, so we can move on with this.  Right now, it is the responsibility of the user to take care and put it in the note).